test/basic.ts

```diff
@@ -102,23 +102,75 @@ test('connection error', async () => {
   expect(isErrorCaught).toBe(true)
 })
 
-test('custom type', async () => {
-  interface User {
+describe('types', () => {
+  type User = {
     username: string
     data: object | null
     age_range: string | null
     status: 'ONLINE' | 'OFFLINE'
     catchphrase: 'string' | null
   }
 
-  // TODO: Find a cleaner way to weave a custom type
-  // eq should show User's properties in LSP/IntelliSense
-  const { data: users } = <{ data: User[] }>(
-    await postgrest.from<User>('users').select().eq('username', 'supabot')
-  )
-  const user = users[0]
-  // Autocomplete should show properties of user after '.'
-  user.username
+  describe('without schema', () => {
+    test('everything works without any types passed in', async () => {
+      // eq should show User's properties in LSP/IntelliSense
+      const { data: users } = await postgrest.from('users').select().eq('username', 'supabot')
+
+      // Should not error on any property
+      users[0].username
+      users[0].somethingElse
```

Comment:
These are for some reason nullable, but I can't find what in the code actually does it... They weren't nullable before and they aren't in other tests..